dwave/system/samplers/leap_hybrid_sampler.py

```diff
@@ -261,7 +261,7 @@ def _sample_large(self, bqm, **kwargs):
         sampleset = self.solver.sample_bqm(sapi_problem_id, **kwargs).sampleset
 
         # relabel, as of dimod 0.9.5+ this is not blocking
-        mapping = dict(enumerate(bqm.iter_variables()))
+        mapping = dict(enumerate(iter(bqm.variables)))
```

Comment:
you can actually just do `dict(enumerate(bqm.variables))` in this case. `enumerate()` just needs an _iterable_

Reply:
✔️